html/index.html

```diff
@@ -6,68 +7,408 @@
+                                        class="character"
+                                        v-ripple
+                                        :class="{'char-selected': selectedCharacter == index}"
+                                        :id="'char-'+ index"
+                                        :data-citizenid="characters[index].citizenid"
+                                        @click="click_character(index, 'existing')"
+                                >
+                                    <span class="slot-name">
+                                        {{characters[index].charinfo.firstname}}{{characters[index].charinfo.lastname}}
+                                        <span class="cid">{{characters[index].citizenid}}</span>
+                                    </span>
+                                </div>
+                            </template>
+                        </div>
+                        <div class="character-btns" v-if="selectedCharacter !== -1">
+                            <v-btn large class="character-btn" id="play" v-ripple @click="play_character">
+                                {{characters[selectedCharacter] !== undefined ? "Play" : "Create Character"}}
+                            </v-btn>
+                            <v-btn large class="character-btn" id="delete" v-if="characters[selectedCharacter] !== undefined && allowDelete == true" v-ripple @click="delete_character">
+                                Delete character
+                            </v-btn>
+                        </div>
                     </div>
-                </div>
 
-                <div class="character-register">
-                    <div class="characters-register-block-header">
-                        <p>Character Registration</p>
-                        <span class="material-symbols-outlined" id="close-reg">close</span>
-                    </div>
-                    <div class="char-register-inputs">
-                        <input type="text" id="first_name" placeholder="First Name" class="char-reg-input">
-                        <input type="text" id="last_name" placeholder="Last Name" class="char-reg-input">
-                        <input type="text" id="nationality" placeholder="Nationality" class="char-reg-input">
-                        <input type="date" id="birthdate" placeholder="Birth Date" min="1900-01-01" max="2999-12-31" class="char-reg-input">
-                        <select id="gender" name="gender">
-                            <option value="Male" selected>Man</option>
-                            <option value="Female">Woman</option>
-                        </select>
+                    <div class="character-info" :class="{'slide-top-right-fade-leave-active': show.characters == false, 'slide-top-right-fade-leave-to': show.characters == false}">
+                        <span class="char-info-titel">Character Information</span>
```

Comment:
```suggestion
                        <span class="char-info-header">Character Information</span>
```
Change along with the CSS